packages/graphql-server-hapi/src/hapiApollo.test.ts

```diff
@@ -1,10 +1,10 @@
 import * as hapi from 'hapi';
-import { graphqlHapi, graphiqlHapi, HapiPluginOptions } from './hapiApollo';
+import { graphqlHapi, graphiqlHapi } from './hapiApollo';
 import 'mocha';
 
 import testSuite, { Schema } from 'graphql-server-integration-testsuite';
 
-function createApp(createOptions: HapiPluginOptions) {
+function createApp(createOptions) {
```

Comment:
Hi DxCx,
createOptions can be either a HapiPluginOptions or GraphiQLPluginOptions now. My options were to either create a new type just for this test function that contained a graphqlOptions and graphiqlOptions or remove the type.

Is there another option that I'm missing? My typescript is pretty weak.